docs/releases/v2018.2.rst

```diff
@@ -130,6 +130,10 @@ Site changes
 No changes need to be made to *site.conf* or *site.mk* when upgrading from
 Gluon v2018.1.x.
 
+Only the optional *opkg* configuration might need an update. The LEDE
+repository setting is ignored and a different OpenWrt repository need to set
```

Comment:
"need to set" sounds wrong, maybe "needs to be set" ?

Reply:
Not sure, maybe something like:
> If an opkg repository for ``lede`` was configured the key needs to be migrated
> to ``openwrt``. ``lede`` is ignored and without an ``openwrt`` key the default
> OpenWrt repository is used. No other changes need to be made to *site.conf*
> or *site.mk* when upgrading from Gluon v2018.1.x.

Reply:
Put 

> No other changes need to be made ...

on a new line, so it can be spotted without reading the paragraph.